testcase.py

```diff
@@ -662,29 +662,27 @@ def get_results(self, point_name, start_time, final_time):
             message = "Invalid value {} for parameter final_time. Value must be a float, integer, or string able to be converted to a float, but is {}.".format(final_time, type(final_time))
             logging.error(message)
             return status, message, payload
-        payload = []
+        payload = {}
         try:
-            # Get correct point
-            if point_name in self.y_store.keys():
-                payload = {
-                    'time': self.y_store['time'],
-                     point_name: self.y_store[point_name]
-                }
-            elif point_name in self.u_store.keys():
-                payload = {
-                    'time': self.u_store['time'],
-                     point_name: self.u_store[point_name]
-                }
-            else:
-                status = 400
-                message = "Invalid value {} for parameter point_name.  Check lists of inputs and measurements.".format(point_name)
-                logging.error(message)
-                return status, message, None
-
+            for point_name in point_names:
+                # Get correct points
+                if point_name in self.y_store.keys():
+                    payload[point_name] = self.y_store[point_name]
+                elif point_name in self.u_store.keys():
+                    payload[point_name] = self.u_store[point_name]
+                else:
+                    status = 400
+                    message = "Invalid value {} for parameter point_names.  Check lists of inputs and measurements.".format(point_names)
```

Comment:
Maybe printing `point_name` (instead of `point_names`) is more useful here. This would tell the user directly which variable does not match the list of inputs and outputs.

Reply:
Good point, thanks.  Done in https://github.com/ibpsa/project1-boptest/pull/480/commits/b79c9c408666e3d0db80187fd12f6978c9ad8f4d.